cms/djangoapps/contentstore/views/course.py

```diff
@@ -1085,6 +1105,7 @@ def settings_handler(request, course_key_string):
                 'upload_asset_url': upload_asset_url,
                 'course_handler_url': reverse_course_url('course_handler', course_key),
                 'language_options': settings.ALL_LANGUAGES,
+                'all_available_secondary_organizations': available_secondary_orgs,
```

Comment:
```suggestion
                'available_secondary_organizations': available_secondary_orgs,
```
???
It's just that the name seems too long 😅